airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx

```diff
@@ -176,23 +158,35 @@ const WebHookForm: React.FC<WebHookFormProps> = ({
           <InputRow>
             <Cell flex={1}>
               <Field name="sendOnFailure">
-                {({ field }: FieldProps<string>) => (
+                {({ field }: FieldProps<boolean>) => (
                   <LabeledToggle
-                    {...field}
-                    name="sendOnFailure"
-                    checked={webhook.sendOnFailure}
+                    name={field.name}
+                    checked={field.value}
+                    onChange={(event) => {
+                      onSubmit({
```

Comment:
Actually I do not think we need to send it on every change. Shouldn't we change only when submit button is pressed? If yes, let's just change it to modify formik state

Reply:
Ok, done.